GetIntoTeachingApi/Controllers/MailingListController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using GetIntoTeachingApi.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Swashbuckle.AspNetCore.Annotations;
+using System;
+using System.Collections.Generic;
+
+namespace GetIntoTeachingApi.Controllers
+{
+    [Route("api/mailing_list")]

```

Comment:
Conventionally the 'api/' part isn't added to each controller.  But it can be sorted later

Reply:
I was going by this (though I now notice they use `RoutePrefix` and not `Route`, but I don't think there's a difference?:

https://docs.microsoft.com/en-us/aspnet/web-api/overview/web-api-routing-and-actions/attribute-routing-in-web-api-2#route-prefixes

Is there somewhere else I can put a route prefix for all the controllers?